vimrc

```diff
@@ -50,11 +45,8 @@ Bundle 'tpope/vim-rails'
 Bundle 'burnettk/vim-angular'
 Bundle 'othree/html5.vim'
 Bundle 'quentindecock/vim-cucumber-align-pipes'
-Bundle 'vim-scripts/vim-terraform'
-Bundle 'rodjek/vim-puppet'
 
 " misc
-Bundle 'tpope/vim-unimpaired'
```

Comment:
i use this one a bunch, but i'm happy to put it in my custom if no one else does